zipline/assets/_assets.pyx

```diff
@@ -89,6 +101,8 @@ cdef class Asset:
         self.first_traded = first_traded
         self.auto_close_date = auto_close_date
 
+
+
```

Comment:
I guess we don't have flake8 for cython, but probably don't want these extra lines.